x-pack/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/findings_flyout.tsx

```diff
@@ -97,7 +97,7 @@ export const CisKubernetesIcons = ({
     <EuiFlexItem grow={false}>
       <EuiIcon type={cisLogoIcon} size="xxl" />
     </EuiFlexItem>
-    <EuiFlexItem grow={false}>
+    <EuiFlexItem grow={false} style={{ marginTop: 4 }}>
       <CISBenchmarkIcon type={benchmarkId} name={benchmarkName} />
     </EuiFlexItem>
   </EuiFlexGroup>
```

Comment:
If we try applying `alignItems="center"` to `<EuiFlexGroup/>` that resolve the issue? If we are using display:flex the we should be able to align the `EuiFlexItems` to be centered.

Reply:
oh good call !